projects/plugins/boost/CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.1-beta] - 2022-03-08
+### Added
+- Critical CSS: Added internal support for Cloud Critical CSS.
+- Critical CSS: Added suggested steps when Critical CSS fails due to page errors.
+- General: Added "My Jetpack" page.
```

Comment:
My Jetpack was added(but not activated) on 1.4.0

Reply:
Yep. e22bc6b31c